packages/next/src/index.ts

```diff
@@ -2712,7 +2713,7 @@ async function getServerlessPages(params: {
         '.',
         normalizedEntry === '/' ? '/index' : normalizedEntry
       )}.js`;
-      const globPath = `${path.join('.', entry)}.js`;
+      const globPath = `${path.posix.join('.', entry)}.js`;
```

Comment:
I'm curious if the dot prefix is doing anything here? I tried this locally and it doesn't do anything:

<img width="288" alt="image" src="https://github.com/vercel/vercel/assets/229881/08144788-d44b-4e2f-b9e1-c3cc50febf08">

Reply:
Seems to just strip the leading slash.

Worth noting that on Windows `path.join('.', '\\file.js')` results in `file.js` while `path.posix.join('.', '\\file.js')` results in `\\file.js`, which makes sense.

If `entry` can start with a `/`, then either keep the `join()` or strip the leading slash.

Reply:
I don't know why this was here, but I think it's probably safe to err on the side of leaving it. It will have different behavior for something like `path.posix.join('.', 'my//file.js')` (I don't know if this could happen, but assuming this maybe was added for a reason)